packages/api-client/src/views/Request/RequestSection/RequestAuth/RequestAuth.vue

```diff
@@ -118,7 +120,7 @@ const authIndicator = computed(() => {
  * Currently selected auth schemes on the collection, we store complex auth joined by a comma to represent the array
  * in the string
  */
-const selectedSchemeOptions = computed(() =>
+const selectedSchemeOptions = computed<SecuritySchemeOption[]>(() =>
```

Comment:
This needed to be typed so the generic type inference on the modelValue would work